jakartaee-web-api/pom.xml

```diff
@@ -200,13 +195,6 @@
             <optional>true</optional>
```

Comment:
In the hidden code block above, there is another <excludeDependencies> for the bind-api.  Shouldn't that be removed as well?  I just tried it on my system and it looks extraneous.

Reply:
Done.